pulsar-broker/src/main/java/org/apache/pulsar/broker/systopic/SystemTopicClient.java

```diff
@@ -168,7 +168,10 @@
     }
 
     static boolean isSystemTopic(TopicName topicName) {
-        return EventsTopicNames.NAMESPACE_EVENTS_LOCAL_NAME.equals(topicName.getLocalName());
+        String localName = topicName.getLocalName();
```

Comment:
what about 
`topicName.getLocalName().startsWith(EventsTopicNames.NAMESPACE_EVENTS_LOCAL_NAME);`

Reply:
+1
```
return topicName != null && topicName.getLocalName().startsWith(EventsTopicNames.NAMESPACE_EVENTS_LOCAL_NAME);
```

Reply:
or
```
return EventsTopicNames.NAMESPACE_EVENTS_LOCAL_NAME.equals(topicName.getPartitionedTopicName());
```

Reply:
> or
> 
> ```
> return EventsTopicNames.NAMESPACE_EVENTS_LOCAL_NAME.equals(topicName.getPartitionedTopicName());
> ```

@315157973  topicName.getPartitionedTopicName() method will return topic name with prefix, such as, `persistent://public/defualt/__change_events`, it will always return `false`.

Reply:
TopicName.get(topicName.getPartitionedTopicName()).getLocalName()